ActionGame/ActionGame/Script/Scene/GameScene.cpp

```diff
@@ -15,22 +17,21 @@ void GameScene::Execute() {
 }
 
 void GameScene::Control() {
-	Player::Move();
-
 	if ( fadeMode != FadeMode::None ) return;
 
+	GameManager::GetInstance()->GetPlayerData()->Move();
```

Comment:
他のクラスに手を突っ込んで掘り返さず、委譲する習慣を付けてください。

例えば
```suggestion
	GameManager::GetInstance()->Control();
```
のような感じで、ドットやアローの数珠つなぎは極力避けましょう。
「知り合いの知り合いは他人」と思うように。

Reply:
了解です。
どこかでついでに修正しておきます。